recipe/meta.yaml

```diff
@@ -51,24 +50,22 @@ requirements:
     - pybind11 <2.11
     - pcre
     - gflags
-    - cyrus-sasl =2.1.27
     - msgpack-c
     - aws-sdk-cpp
     - azure-core-cpp
     - azure-identity-cpp
     - azure-storage-blobs-cpp
     - bitmagic
-    - krb5 =1.20.1
     - xxhash
     - semimap
     - grpcio-tools
     - protobuf
     - spdlog
     - robin_hood
     - recycle
-    - lmdb ==0.9.22
```

Comment:
It would be better to keep the pinning in the reciper rather than in the `conda_config_build_config.yml` file. If this is not possible (for any reason), please add a comment above this line, mentionning that there is still a pinning in the config file.

Reply:
This was tried in 8be20ec but the rerendering then took the globally pinned version of LMDB, see 8c35697.

The specification for LMDB was reverted in 492bfc5.